### YamlMime:ManagedReference
items:
- uid: System.IO.IsolatedStorage.IsolatedStorageException
  id: IsolatedStorageException
  children:
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException
  fullName: System.IO.IsolatedStorage.IsolatedStorageException
  type: Class
  summary: "A exceção é acionada quando ocorre uma falha de uma operação no armazenamento isolado."
  remarks: "`IsolatedStorageException`utiliza o ISS_E_ISOSTORE HRESULT, que tem o valor 0x80131450.       Os exemplos de exceções de armazenamento isolado mais comuns são os seguintes.      -Provas em falta. Armazenamento isolado requer provas (informações sobre a assemblagem e de origem) para determinar a identidade do código e ligá-lo para o espaço de ficheiro associado correto. Sem esta informação, não é possível utilizar armazenamento isolado.      -Operações inválidas. Alguns <xref:System.IO.FileStream>operações não são suportadas para o armazenamento isolado.</xref:System.IO.FileStream>       Para obter uma lista de valores de propriedade inicial para uma instância de `IsolatedStorageException`, consulte o <xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>construtores.</xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class IsolatedStorageException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  id: '#ctor'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> classe com propriedades predefinidas."
  remarks: "A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de mensagem de erro localizado. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException ();
    parameters: []
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> classe com uma mensagem de erro especificado."
  remarks: "A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explique a razão para a exceção."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> classe com dados serializados."
  remarks: "Este construtor é chamado durante a desserialização para reconstitute o objeto de exceção transmitido através de uma transmissão em fluxo."
  syntax:
    content: protected IsolatedStorageException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "O objeto que contém os dados de objecto serializado."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "As nível contextual das informações sobre a origem ou destino."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> classe com uma mensagem de erro especificado e uma referência a exceção interna que é a causa desta exceção."
  remarks: "Uma exceção que é acionada como um resultado direto de uma exceção anterior deve incluir uma referência a exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade devolve o mesmo valor que é transmitido para o construtor ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A referência a exceção interna. |   | <xref:System.Exception.Message%2A>| A cadeia de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explique a razão para a exceção."
    - id: inner
      type: System.Exception
      description: "A exceção é a causa da exceção atual. Se o `inner` não é o parâmetro <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual num <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que processa a exceção interna."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException.IsolatedStorageException
